### DISCUSSION TITLE
A Debian Sid based Build

### DISCUSSION THREAD

**Discussion by bocan** [docker-mailserver/docker-mailserver#4185] [Q&A] [answered]
I was bored this morning and [built docker-mailserver](https://github.com/bocan/docker-mailserver/pull/1/files) with Debian Sid (unstable) instead of Debian 12.

Before I cause the usual backlash by the very mention of Sid, I stress that I'm not suggesting anyone use it - although **I** might spin it up just to see what will happen.

My question is, is there anything alarming in these version changes?  If semantic versioning is anything to go by, I'm mostly worried about clamav, getmail6, and postfix jumping up by minor versions - and rspamd jumping down.

```
                 debian 12                    sid

amavisd-new   1:2.13.0-3+deb12u1            1:2.13.0-6
clamav        1.0.5+dfsg-1~deb12u1          1.3.1+dfsg-5
dovecot-core  1:2.3.19.1+dfsg1-2.1+deb12u1  1:2.3.21.1+dfsg1-1
fail2ban      1.1.0-1~upstream1             1.1.0-1~upstream1
fetchmail     6.4.37-1                      6.4.39-1
getmail6      6.18.11-2                     6.19.02-1
opendkim      2.11.0~beta2-8+deb12u1        2.11.0~beta2-9.1
opendmarc     1.4.2-2+b1                    1.4.2-5
postfix       3.7.11-0+deb12u1              3.9.0-3
postgrey      1.37-2                        1.37-2.1
pyzor         1:1.0.0-6                     1:1.0.0-7
razor         1:2.85-9                      1:2.85-9+b2
rspamd        3.9.1-1~82f43560f~bookworm    3.8.1-1.1
spamassassin  4.0.0-6                       4.0.1-2

**Comment by casperklein** [comment, marked as answer]
> My question is, is there anything alarming in these version changes

I don't know. Check the changelogs and give it a try 😄 But expect it, to be unstable. Unless you really want the thrill of using the latest versions, Debian Trixie (testing) might also be an option and more stable compared to Sid.

**Reply by bocan** [reply]
My worry was that postfix would be that old as there was some known issue with it.  So - I've given it a go and it _appears_ run run like a dream!  My general experience with Sid is that it's **more** stable so that Trixie, so I might leave it for a day or two and see what I can do about getting the latest version of rspamd on it too.

Thanks for the assistance though!

**Comment by polarathene** [comment]
Semver minor version bumps are fine. The rspamd decrease in minor could be breaking if we have anything that expects fixes/features from 3.9 but I don't have any involvement in rspamd support.

I understand wanting to use newer packages, I've often been keen to move to fedora as a base for that reason but I think a blocker there is rspamd (_maybe support for that has been fixed since I last looked_).
